devito/arch/compiler.py

```diff
@@ -410,9 +412,15 @@ def __init__(self, *args, **kwargs):
                 self.ldflags += ['-march=%s' % platform.march]
         elif platform is M1:
             # NOTE:
-            # -march=native unsupported
-            # openmp unusable
-            pass
+            # This requires to install Aplle's llvm (through brew) to install the openmp
+            # librairies and c++ includes. Once installed, both Apple's default clang and
+            # howebrew clang support openmp. Check that openmnp lib exist.
+            if path.exists('/opt/homebrew/opt/llvm/lib/libomp.dylib'):
```

Comment:
It'd be nice to add a (memoized) `get_clang_path` to `arch/archinfo.py`  and then call it here , just like we do with other paths (e.g., CUDA). Hardcoding a path here is a bit ugly

Reply:
Done